### PR TITLE
Adding SDK documentation for message endpoints

### PR DESCRIPTION
**Pull request by petergarbers**

### CODE REVIEW

File: docs/sdk/chat_history/messages.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+# Messages
+
+`Messages` represent individual messages in a conversation. Messages are associated with [`Sessions`](sessions.md) in a M:1 relationship.
+
+The `SessionID` is a string key that access arbritrary identifies. Related data you'd like to store can be persisted as metadata.
```
Comment (danielchalef) [inline docs/sdk/chat_history/messages.md:5]: think this needs a reread.

File: docs/sdk/chat_history/messages.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+Messages must be added to sessions using [`Memories`](memories.md).
+
+You may not update the content of messages, and can only change the metadata.
```
Comment (danielchalef) [inline docs/sdk/chat_history/messages.md:9]: `Currently, updating message content is not supported. You may, however, update a message's metadata.`

**Review by danielchalef — CHANGES_REQUESTED** [review]

File: docs/sdk/chat_history/messages.md (hunk 2 above)
Comment (petergarbers) [reply docs/sdk/chat_history/messages.md:9]: This is much better

**Review by danielchalef — APPROVED** [review]
lgtm

**Pull request merged**